src/components/productsavailable.js

```diff
@@ -1,59 +1,42 @@
-                                 </ul>
+                                <GroupedItems item1="Fishes" link1={Fishes} item2="Eggs" link2={Eggs} item3="Cattle" link3={Cattle}
+                                              item4="Chicken" link4={Chicken} item5="Ram" link5={Ram} heading="livestock & Poultry"
+                                /> 
                             </Col>
                             <Col className="items" md={3}>
-                                 <ul>
-                                     <h6>Fruits / vegetables</h6>
-                                     <li style={{listStyle: 'none'}}><Link to={Cabbage} style={{color: 'DarkGray'}}>Cabbage</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Lettuse} style={{color: 'DarkGray'}}>Lettuse</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Tomatoes} style={{color: 'DarkGray'}}>Tomatoes</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Oranges} style={{color: 'DarkGray'}}>Oranges</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Plantain} style={{color: 'DarkGray'}}>Plaintain</Link></li>
-                                 </ul>
+                                <GroupedItems item1="Cabbage" link1={Cabbage} item2="Lettuse" link2={Lettuse} item3="Tomatoes" 
+                                              link3={Tomatoes} item4="Oranges" link4={Oranges} item5="Plantain" link5={Plantain} 
+                                              heading="Fruits & Vegetables"
+                                  />
                             </Col>
                             <Col className="items" md={3}>
-                                 <ul>
-                                     <h6>Spices</h6>
-                                     <li style={{listStyle: 'none'}}><Link to={Onions} style={{color: 'DarkGray'}}>Onions</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Ginger} style={{color: 'DarkGray'}}>Ginger</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Garlic} style={{color: 'DarkGray'}}>Garlic</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Pepper} style={{color: 'DarkGray'}}>Pepper</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Curry} style={{color: 'DarkGray'}}>Curry</Link></li>
-                                 </ul>
+                               <GroupedItems item1="Curry" link1={Curry} item2="Garlic" link2={Garlic} item3="Ginger" 
+                                             link3={Ginger} item4="Pepper" link4={Pepper} item5="Onions" link5={Onions} 
+                                             heading="Spices" 
+                                  />
```

Comment:
created arrays of objects for available products like you did for `RowItems` then map over them.